src/helpers/ens.ts

```diff
@@ -1,7 +1,6 @@
 import fetch from 'cross-fetch';
-import { namehash } from '@ethersproject/hash';
-import snapshot from '@snapshot-labs/snapshot.js';
 import gateways from '@snapshot-labs/snapshot.js/src/gateways.json';
+import { getSpaceUri } from '@snapshot-labs/snapshot.js/src/utils';
```

Comment:
We shouldnt import function from snapshot.js src folder, can you use snapshot.utils.getSpaceUri instead?

Reply:
It's like that in many places. I think you could also set the entry point differently or so, to get rid of the `src` part. And isn't the treeshaking approach the perferred one over importing the entire library?